style.css

```diff
@@ -0,0 +1,454 @@
+  padding: 16px;
+  gap: 12px;
+  background: #fff;
+  border: 1px solid #dfe1e6;
+  border-radius: 16px;
+  margin-top: 88px;
+  width: 250px;
+  height: 600px;
+}
+
+.work-image-2 {
+  background-image: url(./assets/Snapshoot\ Portfolio2.svg);
+  height: 220px;
+  border-radius: 8px;
+}
+
+.work-image-3 {
+  background-image: url(./assets/Snapshoot\ Portfolio3.svg);
+  height: 220px;
+  border-radius: 8px;
+}
+
+.work-image-4 {
+  background-image: url(./assets/Snapshoot\ Portfolio4.svg);
+  height: 220px;
+  border-radius: 8px;
+}
+
+.about-myself {
+  position: absolute;
+  width: 300px;
+  height: 1263px;
+  margin-top: 144px;
+  border-top-right-radius: 100px;
+  background: #fff;
+}
+
+.about-title {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  color: #172b4d;
+  width: 327px;
+  height: 101px;
+  display: flex;
+  align-items: center;
+  margin: 0;
+  margin-top: 114px;
+}
+
+.des {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #344563;
+}
+
+.resume {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 12px;
+  gap: 10px;
+  border: 1px solid #6070ff;
+  border-radius: 8px;
+  background: #fff;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  color: #396df2;
+  margin-top: 50px;
+}
+
+.language {
+  position: absolute;
+  height: 24px;
+  left: 12px;
+  right: 78px;
+  color: #000;
+  display: flex;
+  align-items: center;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 20px;
+  line-height: 24px;
+  justify-content: space-between;
+  width: 250px;
+  margin-top: 20px;
+}
+
+.language-list {
+  display: flex;
+  flex-direction: column;
+  padding: 12px;
+  gap: 12px;
+  margin-top: 36px;
+}
+
+.js {
+  background-image: url(./assets/Ellipse\ 1.svg);
+  border-radius: 8px;
+  width: 303px;
+  height: 66.72px;
+  margin-top: 36px;
+}
+
+.lang-list {
+  list-style: none;
+  padding: 0;
+  flex-direction: column;
+  display: flex;
+  justify-content: space-between;
+}
+
+.language-list ul li {
+  display: flex;
+  align-items: center;
+  color: #253858;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 20px;
+  left: 76px;
+  background: #f7f7f9;
+  border-radius: 8px;
+  margin-bottom: 12px;
+  width: 260px;
+}
+
+.language-list ul li span {
+  margin-left: 16px;
+}
+
+.framework {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 20px;
+  line-height: 24px;
+  color: #000;
+  justify-content: space-between;
+  display: flex;
+  width: 237px;
+  padding: 15px 0;
+}
+
+.line {
+  width: 270px;
+  height: 1px;
+  background: #dfe1e6;
+  display: flex;
+}
+
+body {
+  background: #e5e5e5;
+}
```

Comment:
- [ ] kindly update the color of the background , it should `#F7F7F9` be instead of `#e5e5e5` like shown in figma design